code/__DEFINES/atmospherics/atmos_helpers.dm

```diff
@@ -32,10 +32,10 @@
 
 ///Directly adds a gas to a gas mixture without checking for its presence beforehand, use only if is certain the absence of said gas
 #define ADD_GAS(gas_id, out_list)\
-	var/list/tmp_gaslist = GLOB.gaslist_cache[gas_id]; out_list[gas_id] = tmp_gaslist.Copy();
+	var/static/list/list/_add_gas_gaslist_cache = GLOB.gaslist_cache; out_list[gas_id] = _add_gas_gaslist_cache[gas_id].Copy();
```

Comment:
This name is way too verbose
Just call it _static_gaslist_cache, or even just _gaslist_cache